src/grc65/main.c

```diff
@@ -850,8 +850,12 @@ static char *filterInput (FILE *F, char *tbl)
     /* loads file into buffer filtering it out */
     int a, prevchar = -1, i = 0, bracket = 0, quote = 1;
 
-    for (;;) {
-        a = getc(F);
+    a = getc(F);
+    while (1)
+    {
```

Comment:
please put the opening brace after the while()

Reply:
[1df7ab0](https://github.com/cc65/cc65/pull/2098/commits/1df7ab0352a43546c6ece76830594a687681c938)